Developers/Unity Project/Spooksville/Assets/Scripts/Items/WeaponManager.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WeaponManager : MonoBehaviour
+{
+    public static WeaponManager instance;
+    private List<Weapon> weapons = new List<Weapon>();
+
+    public Weapon GetRandomWeapon()
+    {
+        return weapons[Mathf.RoundToInt(Random.Range(0, weapons.Count - 1))];
+    }
+
+    void Start()
+    {
+        if (instance == null)
+        {
+            instance = this;
+            DontDestroyOnLoad(gameObject);
+        }
+
+        RegisterWeapons();
+
+        foreach (Weapon weapon in weapons)
```

Comment:
I'd recommend not using statement shorthand unless you are putting the entire statement on a single line, i.e. `if (foo != null) foo.bar();`. It's easy for someone to accidentally add a second line that's indented and looks like it's part of the statement, when it's not. This can sometimes happen during a merge and nobody will notice. A linter would help catch issues like this. Even so, it's led to [a horrible security vulnerability](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/) amongst other things.